### ISSUE TITLE
Prometheus Integration

### ISSUE THREAD

**Issue by marcelinhov2** [SoftInstigate/restheart#293]
I tried to plug Prometheus to monitoring my RestHeart API but I'm getting a parse error:

```
level=warn ts=2018-06-21T21:12:52.174717536Z caller=scrape.go:713 component="scrape manager" scrape_pool="General RestHeart" target=http://xx.xx.xx.xx:80/google/_metrics msg="append failed" err="strconv.ParseFloat: parsing \"milliseconds\": invalid syntax"
```

Does anyone know how can I fix it?

Thanks guys!

**Comment by ujibang** [comment]
Hi @lenalebt,

can you help @marcelinhov2 on this?

**Comment by marcelinhov2** [comment]
any update?

Thanks!

**Comment by lenalebt** [comment]
Hi, sorry - somehow that mail got sorted as Spam :(. Yep, here to assist. I need to admit that we're currently using the JSON version and the prometheus version was more like a textbook implementation and direct translation from the JSON version. Having a look at the example I think it's a problem that the units are written as String, and Prometheus only supports numbers - I could have gotten that. I will provide a fix.

**Comment by mkjsix** [comment]
The fix is available on both 3.4.0-SNAPSHOT and 3.3.7-SNAPSHOT.

**Comment by marcelinhov2** [comment]
Great. In time, do you guys have any Grafana Dashboard ready to receive this data?

**Comment by mkjsix** [comment]
Besides 3.4.0-SNAPSHOT and 3.3.8-SNAPSHOT, this fix is now available in stable release RESTHeart [3.3.7](https://github.com/SoftInstigate/restheart/releases/tag/3.3.7).

As usual, Docker images are available [here](https://hub.docker.com/r/softinstigate/restheart/tags/).

**Comment by mkjsix** [comment]
@marcelinhov2 about Grafana: sorry but we don't have any dashboard. If you find / build anything please let us know.
